README.md

```diff
@@ -37,6 +38,9 @@ optional arguments:
                         input manifest (optional)
   --disable-genome-cache
                         Disable caching of genome reference data
+  --include-attributes [{BAF,LRR} [{BAF,LRR} ...]]
```

Comment:
I don't think any of these attributes are required by specification, so could also include GT, GQ here as well (with potentially GT, GQ as a default)